static/app/views/performance/queues/destinationSummary/messageSpanSamplesPanel.tsx

```diff
@@ -333,6 +359,22 @@ export function MessageSpanSamplesPanel() {
             />
           </ModuleLayout.Full>
 
+          <ModuleLayout.Full>
+            <Feature features="performance-sample-panel-search">
```

Comment:
Same note about nesting `ModuleLayout.Full` inside `Feature` rather than the other way around